governance/votes/Executive vote - May 14, 2021.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Template - [Executive Vote] Parameter Changes and Oracle Updates - May 14, 2021
+
+---
+# [Executive Proposal] Parameter Changes and Oracle Updates - May 14, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+### Parameter Changes
```

Comment:
Honestly in this case I would just remove the headings here. I only tend to use them when there are significant amounts of different content.